_dev/src/ts/routing/RouteHandler.ts

```diff
@@ -0,0 +1,42 @@
+export default class RouteHandler {
+  constructor() {
+    window.AutoUpgrade.classes.RouteHandler = this;
```

Comment:
Is this a good practice? Shouldn't it be better for the caller of this class to be responsible in defining it in the global domain?